dev_requirements.txt

```diff
@@ -1,9 +1,6 @@
-dbt-core==0.19.0
-pyodbc>=4.0.27
-azure-identity>=1.4.0
 black~=20.8b1
```

Comment:
I renamed it to `dev_requirements` since it are the packages you need to install for local development. Dbt also has a `dev_requirements.txt`